conftest.py

```diff
@@ -0,0 +1,114 @@
+import pytest
+import datetime
+from django.utils import timezone
+from message.models import Message, AuthorOptions
+from daycare.models import DayCare, Image
+from dogowner.models import DogOwner
+from orders.models import Order
+from review.models import Review
+
+
+@pytest.fixture
+def create_order():
+    return Order.create(dog_owner_id=DogOwner.objects.get(id=1),
+                        daycare_id=DayCare.objects.get(id=1),
+                        start_date=timezone.now(),
+                        end_date=timezone.now() + datetime.timedelta(days=3),
+                        price_per_day=100,
+                        )
+
+
+@pytest.fixture
+def dogowner_message_to_daycare1():
```

Comment:
maybe `dogowner1_message_to_daycare1`

Reply:
I agree, it was implicated by @Yuval-Vino in message app tests so I want to hear what do you think also

Reply:
all tests use a single dog owner, so I don't see the benefit of IDing the single dogowner, do you see it differently? @ErezCohenn  @jeniaSakirko

Reply:
umm, if it's only one dog-owner I think it can remain as it is right now.

Reply:
Please refrain from changing names in this PR when it is not mandatory. You can open a new PR for that once this one is merged.